components/reset/reset-board.vue

```diff
@@ -0,0 +1,62 @@
+<template>
+  <div class="container-fluid mt-5">
+    <div class="mt-5 row justify-content-center">
+      <div class="mt-5 col-6 position-relative text-light">
+        <div class="card animated fadeInUp bg-dark shadow-lg">
+          <MCSubTitle style="text-align:center; padding:1rem;">
+            {{ sectionTitle }}
+          </MCSubTitle>
+        </div>
+        <br>
+        <div class="mb-5 card animated fadeInUp bg-dark shadow-lg px-5 py-5">
```

Comment:
Use `b-form` instead of `div`?
Checkout the Bootstrap's [form component](https://bootstrap-vue.org/docs/components/form).